packages/admin-next/dashboard/src/routes/price-lists/common/hooks/use-price-list-grid-columns.tsx

```diff
@@ -76,10 +93,21 @@ export const usePriceListGridColumns = ({
         })
       }),
       ...regions.map((region) => {
+        const preference = pricePreferences.find(
+          (p) => p.attribute === "region_id" && p.value === region.id
+        )
+
         return columnHelper.column({
           id: `region-price-${region.id}`,
           name: `Price ${region.name}`,
-          header: `Price ${region.name}`,
+          header: () => (
+            <div className="flex w-full items-center justify-between gap-3">
```

Comment:
Maybe we should make a reusable header component for this. So if we want to change it at some point we can do it from a single place.

Reply:
@kasperkristensen this refers to my second point in the description - I was gonna only extract the header, but it felt like we can extract a lot more (eg. create a PricingGrid component, or something along those lines), so I left it for now until we discuss it.

WDYT, should we extract the entire pricing data grid?

Reply:
I think the header would be a good start, extracting the table in it self would be ideal. 
 
 They will be the same across the project except for two parameters: The way data is instantiated, and whether the data is displayed purely in relation to the product variants or also the owning product (subrows). If you can abstract that away in a nice way then I think it would be be great, but if its a lot of work, I am not sure if its worth it, since the amount of boilerplate for setting up the grids aren't too much, especially if we make the header re-usable.

Reply:
Ok let me see if it can be extracted nicely, if not I'll just do the header.